ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/CommandDeduplicationIT.scala

```diff
@@ -468,6 +458,89 @@ final class CommandDeduplicationIT(
     } yield {}
   })
 
+  testGivenAllParticipants(
```

Comment:
Decided to add it as a separate test case, as the previous one was no longer "basic".